src/counter_analysis_toolkit/flops.c

```diff
@@ -832,6 +832,7 @@ void exec_flops( int precision, int EventSet, FILE *fp ) {
 #endif
           break;
       default:
+          break;
     }
```

Comment:
There should not be any need for break in `default`

Reply:
Should I use an empty statement instead; as in, just a semicolon?

Reply:
Yes that would also work.